backend/app.py

```diff
@@ -3,22 +3,37 @@
 from waitress import serve
 from http import HTTPStatus
 
+from classification.file_loading import get_raw_array
+from classification.predict import predict
+from classification.exceptions import ClassificationError
+from classification.config.constants import Sex, ALLOWED_FILE_EXTENSIONS
 
 app = Flask(__name__)
 
 
 def allowed_file(filename):
-    ALLOWED_EXTENSIONS = {'txt', 'csv'}
-    return '.' in filename and filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+    return '.' in filename and filename.lower().endswith(tuple(ALLOWED_FILE_EXTENSIONS))
```

Comment:
```suggestion
    return filename.lower().endswith(ALLOWED_FILE_EXTENSIONS)
```

et mettre le `.` devant l'extension, car ce code flag `al.lotxt` comme étant valide.